docs/deployment.md

```diff
@@ -0,0 +1,99 @@
+# Deployment
+
+1. Create a Google Cloud Project if you do not have one. Enable the Cloud Build and Container Registry APIs.
+
+2. Set the project and compute zone.
```

Comment:
Probably add "Open a new Cloud Shell in the Cloud console of the project " before listing the shell commands